src/cbor/fallback_decoder.rs

```diff
@@ -268,7 +268,10 @@ impl FallbackDecoder {
 
                 // unwrap is safe, the other side would have to drop for a
                 // panic – can’t happen, because we control it:
-                request.response_tx.send(response).unwrap();
+                request
+                    .response_tx
+                    .send(response)
+                    .unwrap_or_else(|_| unreachable!());
```

Comment:
A little controversial, but IMO `unreachable!()` is cleaner? And conveys the message better. "We’re sure this branch is unreachable, or else kill everything."